### PR TITLE
ACA-4600 Added edit profile link

### PR DESCRIPTION
**Pull request by sbansal3107**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/userinfo/components/user-info.component.html [hunk 1]
```diff
@@ -148,6 +148,7 @@
                                       class="adf-userinfo__detail-title">{{identityUser | fullName}}</span>
                                 <span class="adf-userinfo__detail-profile"
                                       id="identity-email"> {{identityUser.email}} </span>
+                                <a class="adf-userinfo__detail-profile" href="#/profile">My Profile</a>
```
Comment (DenysVuika) [inline lib/core/userinfo/components/user-info.component.html:151]: all the text must be coming from the i18n resources

**Review by DenysVuika — CHANGES_REQUESTED** [review]

File: lib/core/userinfo/components/user-info.component.html (hunk 1 above)
Comment (sbansal3107) [inline lib/core/userinfo/components/user-info.component.html:151]: Route for '/profile' will be covered in this PR https://github.com/Alfresco/alfresco-content-app/pull/2572

**Review by DenysVuika — APPROVED** [review]

File: lib/core/userinfo/components/user-info.component.html (hunk 1 above)
Comment (sbansal3107) [reply lib/core/userinfo/components/user-info.component.html]: Done

**Review by MohinishSah — APPROVED** [review]

**Review by arditdomi — COMMENTED** [review]
Missing unit tests about when this should be visible (In case of ECM, BPM, ALL? ) and also about the location where the link should redirect you

**Pull request merged**